### PR TITLE
UGENE-4985. Wrong error message is shown when trying to open deleted file

### PR DESCRIPTION
**Pull request by evelinabis**
https://ugene.dev/tracker/browse/UGENE-4985

### CODE REVIEW

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_4001_5000.cpp [hunk 1]
```diff
@@ -6030,6 +6030,39 @@ GUI_TEST_CLASS_DEFINITION(test_4983) {
...
 
+GUI_TEST_CLASS_DEFINITION(test_4985) {
+    // Create file _common_data/scenarios/sandbox/A.fa with text "A"
+    // Open this file
+    // Delete this file from hard disk
+    // "File Modification Detected" dialog appears. Click "No"
+    // On Start Page open recent file A.fa
+    //      Expected: error message box with text "File doesn't exist: _common_data\scenarios\sandbox\a.fa" appears
+
+    QFile file(testDir + "_common_data/scenarios/sandbox/A.fa");
```
Comment (yalgaer) [inline src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_4001_5000.cpp]: Use 1 liner like: Use IOAdapterUtils::writeTextFile(sandBoxDir + "test_6707/file.txt", "Hello!");

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_4001_5000.cpp [hunk 2]
```diff
@@ -6030,6 +6030,39 @@ GUI_TEST_CLASS_DEFINITION(test_4983) {
...
+    file.open(QIODevice::WriteOnly | QIODevice::Text);
+    QTextStream out(&file);
+    out << 'A';
+    out.flush();
+    CHECK_SET_ERR(file.size() == 1, "Error writing file " + file.fileName())
+
+    GTFileDialog::openFile(os, file.fileName());
+    GTUtilsTaskTreeView::waitTaskFinished(os);
```
Comment (yalgaer) [inline src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_4001_5000.cpp]: Check that something is changed: an object view is opened or project tree has the document.

File: src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_4001_5000.cpp [hunk 3]
```diff
@@ -6030,6 +6030,39 @@ GUI_TEST_CLASS_DEFINITION(test_4983) {
...
+    GTUtilsDialog::waitForDialog(os, new MessageBoxNoToAllOrNo(os));
+    GTUtilsStartPage::openStartPage(os);
+    GTWidget::click(os, GTWidget::findLabelByText(os, "- A.fa").first());
+
+    QMessageBox *dialog = qobject_cast<QMessageBox *>(GTWidget::getActiveModalWidget(os));
+    CHECK_SET_ERR(dialog != nullptr, "Expected MessageBox")
+    QString actual = dialog->text();
+    QString expected = "File doesn't exist: " + QFileInfo(file.fileName()).absoluteFilePath();
+    CHECK_SET_ERR(actual == expected, "MessageBox text: expected \"" + expected + "\", actual \"" + actual + '"')
+    dialog->button(QMessageBox::Ok)->click();
```
Comment (yalgaer) [inline src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_4001_5000.cpp]: Can we use: GTUtilsDialog::waitForDialog(os, new MessageBoxDialogFiller(os, QMessageBox::Ok, "text")); here?
If will fail if there is no message box with the expected text.

**Review by yalgaer — CHANGES_REQUESTED** [review]

**Comment by yalgaer** [comment]
The fix is good, only GUI test can be shortened by using correct utility methods.

**Review by smlnkcv — COMMENTED** [review]
Add please Russian translation for error message

**Review by yalgaer — APPROVED** [review]

**Pull request merged**